mathics/main.py

```diff
@@ -242,6 +242,7 @@ def main() -> int:
 
     argparser.add_argument(
         "--execute",
+        "-c",
```

Comment:
In WolframScript, "-c" has the meaning of '-e'

Reply:
Ok, removed then. I am sure though I saw someone use "-c" some place.